ebms-payload/src/main/kotlin/no/nav/emottak/payload/crypto/PayloadSignering.kt

```diff
@@ -14,11 +17,33 @@ import javax.xml.crypto.dsig.dom.DOMSignContext
 import javax.xml.crypto.dsig.spec.C14NMethodParameterSpec
 import javax.xml.crypto.dsig.spec.TransformParameterSpec
 
-private fun payloadSigneringConfig() = FileKeyStoreConfig(
-    keyStoreFilePath = getEnvVar("KEYSTORE_FILE", "xml/signering_keystore.p12"),
-    keyStorePass = getEnvVar("KEYSTORE_PWD", "123456789").toCharArray(),
-    keyStoreType = getEnvVar("KEYSTORE_TYPE", "PKCS12")
-)
+private fun payloadSigneringConfig() =
```

Comment:
**Spørsmål og et forslag basert på om det tilhører ebms-payload (du velger selv):** 

Vi har `payloadSigneringConfig`, og vi har `dekrypteringConfig`. Tilhører `dekrypteringConfig` ikke til ebms-payload? Hvis så, hvorfor heter den ikke `payloadDekrypteringConfig`? Evt. ta vekk `payload` fra `payloadSigneringConfig` hvis begge uansett tilhører `ebms-payload`, siden det kan virke overflødig. Det blir vel ikke et tilfelle vi må referere til disse to utenfra `ebms-payload`